README.md

```diff
@@ -22,7 +22,6 @@ DoubleEntry uses the [Money gem](https://github.com/RubyMoney/money) to encapsul
 DoubleEntry is tested against:
 
 Ruby
- * 2.0.0
  * 2.1.5
```

Comment:
From the travis file it looks like we start at 2.1.7: maybe we should update this